sub-projects/device/utils/compare.py

```diff
@@ -0,0 +1,42 @@
+import boto3
+import json
+from PIL import Image
+
+class Rekgoniton(object):
```

Comment:
Move this file to `sub-projects/device/src/` and change file name to `AWS` (because we're dealing with AWS stuff in this file).